fixtures/react_native_windows_e2e_app/babel.config.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  presets: [['module:../../packages/haul-babel-preset-react-native', { chakra: true }]],
```

Comment:
Is there any specific reason why switching to Metro's Babel preset??

Reply:
It comes with default RNW project template. Let me test Haul preset and fix it.

Reply:
Fixed. Thanks for the catch :)